# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libjpeg-turbo/hotfix-cpu-target.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Fix cmake cpu detection.
  - NoTag <notag@maneto.fr>

--- libjpeg-turbo-2.1.4/CMakeLists.txt.vanilla	2023-04-24 17:09:08.517758745 +0200
+++ libjpeg-turbo-2.1.4/CMakeLists.txt	2023-04-24 17:09:45.508756808 +0200
@@ -55,6 +55,9 @@
 
 include(cmakescripts/PackageInfo.cmake)
 
+# allow us to override the arch with a flag because it complains on auto-detect
+option(CMAKE_SYSTEM_PROCESSOR "Cpu target architecture override" "foobar")
+
 # Detect CPU type and whether we're building 64-bit or 32-bit code
 math(EXPR BITS "${CMAKE_SIZEOF_VOID_P} * 8")
 string(TOLOWER ${CMAKE_SYSTEM_PROCESSOR} CMAKE_SYSTEM_PROCESSOR_LC)
